Makefile.am

```diff
@@ -637,10 +637,12 @@ mjr2pcap_SOURCES = \
 mjr2pcap_CFLAGS = \
 	$(AM_CFLAGS) \
 	$(POST_PROCESSING_CFLAGS) \
+	$(BORINGSSL_CFLAGS) \
```

Comment:
Why BORINGSSL? That's not wat we use in configure as a variable for the crypto stuff.

Reply:
uh, that was just my naive attempt to fix the build after grepping for "ssl" and mjr2pcap... where do I have to fix this for real?

Reply:
It's probably easier to add the ssl and/or crypto checks to the `POST_PROCESSING` definition in `configure.ac`, which will make it available to all post-processing tools. The `BORINGSSL_CFLAGS` and `BORINGSSL_LIBS` variables are still needed in case BoringSSL is used: you can refer to how `janus_CFLAGS` and `janus_LDADD` use them.

Reply:
Sorry, it seems I cannot get it to work. I added `libssl >= $ssl_version` to `POST_PROCESSING`, but the build still fails. I don't know what else I have to do. I have used CMake, but not autotools before :grimacing: